src/components/Footer/index.tsx

```diff
@@ -91,7 +93,9 @@ const Footer = () => {
     <section className={styles.FooterContainer} data-testid='footer-text'>
       <div className={styles.FooterBody}>
         <div className={styles.LogoWrapper}>
-          <img src='img/gray-logo.svg' alt='Deriv API Logo' className={styles.FooterLogo} />
+          <div className={styles.FooterLogo}>
+            <GrayLogo />
```

Comment:
does this implementation support Alt text behavior by defualt ? Like if an image got crash for any reason does it show alt txt or something ?

Reply:
No I guess, the SVG works similarly to a react component, which mean it won't break 🤣